apps/web/src/screens/Dashboard/components/AptosTokensTab/Collection.tsx

```diff
@@ -0,0 +1,53 @@
+import type { FC } from 'react';
+import { Image, StyleSheet } from 'react-native';
+import { Hoverable, Text } from '@walless/gui';
+
+export interface AptosCollection {
+	id: string;
+	name: string;
+	uri: string;
+	itemCount: number;
+}
+
+interface Props {
+	item: AptosCollection;
+	onPress?: (id: string) => void;
+}
+
+const Collection: FC<Props> = ({ item, onPress }) => {
+	return (
+		<Hoverable
+			style={styles.container}
+			onPress={onPress && (() => onPress(item.id))}
+		>
+			<Image style={styles.image} source={{ uri: item.uri }} />
+			<Text style={styles.text}>
+				{item.name} ({item.itemCount})
+			</Text>
+		</Hoverable>
+	);
+};
+
+export default Collection;
+
+const styles = StyleSheet.create({
+	container: {
+		width: 158,
```

Comment:
check if this hardcode width is necessary

Reply:
this `AptosTokensTab/Collection.tsx` is a clone of `apps/web/src/screens/Dashboard/components/CollectibleItem.tsx`, the hard-coded `style` is retrieved from there